library.json

```
@@ -1,27 +1,48 @@
 {
   "$schema": "https://raw.githubusercontent.com/platformio/platformio-core/develop/platformio/assets/schema/library.json",
-  "name": "LibraryTemplate",
+  "name": "Min-heap",
   "version": "1.0.0",
-  "description": "Templates for libraries compatible with the PlatformIO ecosystem",
+  "description": "This library implements all the functions used to create and manage a minimum heap data structure. Can be used for any embedded system.",
   "keywords": [
-    "template"
+    "min-heap",
+    "heap",
+    "embedded"
   ],
   "repository": {
     "type": "git",
-    "url": ""
+    "url": "https://github.com/eagletrt/libmin-heap-sw"
   },
   "authors": [
     {
       "name": "Antonio Gelain",
       "email": "antonio.gelain2@gmail.com",
       "maintainer": true
+    },
+    {
+      "name": "Dorijan Di Zepp",
+      "email": "dorijan.dizepp@eagletrt.it",
+      "maintainer": true
     }
   ],
   "license": "AGPL-3.0-only",
   "frameworks": "*",
   "platforms": "*",
-  "headers": [],
-  "examples": [],
+  "dependencies": [
+    {
+      "name": "ArenaAllocator",
+      "version": "https://github.com/eagletrt/libarena-allocator-sw.git"
```

Tag version should be specified when using a dependency.
```suggestion
      "version": "https://github.com/eagletrt/libarena-allocator-sw.git#v1.0.0"
```